api/tests/test_authenticate.py

```diff
@@ -59,6 +63,18 @@ def test_authenticate_with_signature_exception_returns_401(mocker, app):
 
         mock_abort.assert_called_once_with(401)
 
+def test_authenticate_with_valid_refresh_token(mocker, app):
```

Comment:
nit-pick: I believe it would be slightly more informative to state it like "test authenticate with expired token" or something along those lines (we are not testing the validity of the refresh token)

Reply:
Nice, I'll follow your advice